descartes_light/test/utest.cpp

```diff
@@ -225,6 +262,57 @@ TYPED_TEST(SolverFixture, KnownPathTest)
   }
 }
 
+TYPED_TEST(ParameterizedSolverFixture, ParameterizedKnownPath)
```

Comment:
I think the value of this test is less in checking that the solvers work for all of these test cases and more in the collection of benchmark data to compare the solvers. Things that can easily be tracked for all solvers might be:
- Amount of time required to build the graph
- Amount of time required to solve the graph
- The size in memory of the solver object once it contains the fully constructed graph

I think we should amend this PR to calculate these things and at least print them out to the terminal if we don't save them in some other way.

I think there will also be a lot of interesting things we can check about BGL implementations specifically that would require a different test fixture specifically for BGL solvers (number of nodes that were visited/opened by the search, number of edge costs that were calculated during the search, etc.). We can implement that test later, but the point is that we should be collecting data about solver performance in addition to running the tests

Reply:
I think the first two should be easy to add, I'm not sure if computing the sized of the constructed graph is currently supported but I can look into that and add that feature if the BGL supports it.

Reply:
I agree on the long term vision with bgl introspection, but I'm comfortable saving that for a later PR if we have the other metrics implemented